docs/src/pages/demos/lists/lists.md

```diff
@@ -26,6 +26,10 @@ Lists are best suited for similar data types.
 
 {{demo='pages/demos/lists/NestedList.js'}}
 
+### Pinned Subheader List
+
+{{demo='pages/demos/lists/PinnedSubheaderList.js'}}
```

Comment:
Please add some remark around the browser support here.

Reply:
You can also say that browser that doesn't support it will render as before, the UI won't be broken.